src/test/scala/biowdl/test/TestPipelineFunctional.scala

```diff
@@ -21,8 +21,7 @@
 
 package biowdl.test
 
-import java.io.File
-
-import nl.biopet.utils.biowdl.PipelineSuccess
-
-trait TestPipelineSuccess extends TestPipeline with PipelineSuccess
+import nl.biopet.utils.biowdl.annotations.Ensembl87
+class TestPipelineFunctional extends CompareGffSuccess with Ensembl87 {
```

Comment:
This test still doesn't have an input sample, it will probably fail.

We can probably drop the functional tests here, this can be tested in biowdl/RNA-seq